plugins/clickhouse/include/clickhouse/arguments.hpp

```diff
@@ -74,7 +74,37 @@ struct operator_arguments {
     parser.named("primary", primary_selector, "field");
     res.ssl.add_tls_options(parser);
     TRY(parser.parse(inv, ctx));
-    if (not validate_identifier(res.table.inner)) {
+    const auto quoting = detail::quoting_escaping_policy{.quotes = "\""};
+    const auto dot = quoting.find_first_of_not_in_quotes(res.table.inner, ".");
```

Comment:
Please resolve this after testing that quoting and the error paths work as expected.